docs/api-endpoints.md

```diff
@@ -0,0 +1,301 @@
+```
+
+- Parameters
+
+  | Name                | Type                      | In    | Description                               |
+  | ------------------- | ------------------------- | ----- | ----------------------------------------- |
+  | `environment`       | 'dev' \| 'test' \| 'prod' | path  | The environment to search the users in    |
+  | `id`                | number                    | path  | The id of the integration                 |
+  | `search` (optional) | string                    | query | The search key                            |
+  | `first` (optional)  | number                    | query | The pagination offset (defaults to 0)     |
+  | `max` (optional)    | number                    | query | The maximum results size (defaults to 50) |
+
+- Responses
+
+<table style="margin-left: 2em;">
+<tr><td>Status Code</td><td>Response</td></tr>
+<tr>
+<td>200</td>
+<td>
+
+```json
+["admin", "manager"]
+```
+
+</td>
+</tr>
+<tr>
+<td>422</td>
+<td>
+
+```json
+{ "success": false, "message": "<error_message>" }
+```
+
+</td></tr>
+</table>
+
+#### Get all roles for the integration user
+
+```
+GET /api/{environment}/integrations/{id}/users/{username}/roles
+```
+
+- Parameters
+
+  | Name          | Type                      | In   | Description                            |
+  | ------------- | ------------------------- | ---- | -------------------------------------- |
+  | `environment` | 'dev' \| 'test' \| 'prod' | path | The environment to search the users in |
+  | `id`          | number                    | path | The id of the integration              |
+  | `username`    | string                    | path | The username of the user               |
+
+- Responses
+
+<table style="margin-left: 2em;">
+<tr><td>Status Code</td><td>Response</td></tr>
+<tr>
+<td>200</td>
+<td>
+
+```json
+["admin", "manager"]
+```
+
+</td>
+</tr>
+<tr>
+<td>422</td>
+<td>
+
+```json
+{ "success": false, "message": "<error_message>" }
+```
+
+</td></tr>
+</table>
+
+#### Manage the user roles
+
+```
+PUT /api/{environment}/integrations/{id}/users/{username}/roles
+```
+
+- Parameters
+
+  | Name            | Type                      | In   | Description                            |
+  | --------------- | ------------------------- | ---- | -------------------------------------- |
+  | `environment`   | 'dev' \| 'test' \| 'prod' | path | The environment to search the users in |
+  | `id`            | number                    | path | The id of the integration              |
+  | `username`      | string                    | path | The username of the user               |
+  | `rolesToAdd`    | string[]                  | body | array of roles to add                  |
+  | `rolesToRemove` | string[]                  | body | array of roles to remove               |
+
+- Responses
+
+<table style="margin-left: 2em;">
+<tr><td>Status Code</td><td>Response</td></tr>
+<tr>
+<td>200</td>
+<td>
+
+```json
+// all roles for the user after the changes
```

Comment:
same as above

Reply:
I moved the comment out of the json block.

Reply:
thank you